src/main/java/com/ecwid/maleorang/method/v3_0/lists/StatsInfo.kt

```diff
@@ -59,7 +59,7 @@ class StatsInfo : MailchimpObject() {
 
     @JvmField
     @Field
-    var open_rate: Int? = null
+    var open_rate: Double? = null
```

Comment:
I see some other "rate" fields in this class. Shouldn't they all be converted to Doubles?